.github/workflows/ci_pipeline.yml

```diff
@@ -108,7 +180,7 @@ jobs:
           source /opt/spack/share/spack/setup-env.sh
           spack load libpressio
            
-          coverage run -m pytest --continue-on-collection-errors -v --durations=0 pySDC/tests -m libpressio
+          coverage run -m pytest --continue-on-collection-errors -v --durations=0 pySDC/projects/compression/tests
```

Comment:
Let's not remove the pytest marker

Reply:
We can also use the markers. I chose this change to make sure to include all tests there; also ones, where the marker may be missing. But We can also include the requirement for a marker there, if you prefer that.

Reply:
Yes please. The marker marks tests that should be run in the docker container. Let's use it like it's intended.

Reply:
Readded the marker in b858c33c7fdec93b049845f5232f5b817c127c27